src/services/widgets.data.service.ts

```diff
@@ -8,47 +8,39 @@ import { getVerifiedWidgetsData } from './data.verification/data.verification.se
 const NEWS_FLASH_API: string = '/api/infographics-data';
 const WIDGETS_BY_LOCATION_API: string = 'api/infographics-data-by-location';
 
-export const fetchWidgets = async (id: number, lang: string, yearAgo?: number): Promise<ILocationData | undefined> => {
-  if (showDemoCards && id === DEMO_ID) {
-    return getDemoWidgetData();
+export interface IWidgetInput {
+  lang: string;
+  newsId?: number;
+  gpsId?: number;
+  yearAgo?: number;
+}
```

Comment:
To prevent mistakes, we better make `itemId` mandatory and add `LocationSource` enum 
```suggestion

enum LocationSourceType {
  newsFlash,
  gps
}

export interface IWidgetInput {
  lang: string;
  itemId: number;
  itemType: LocationSourceType;
  yearAgo?: number;
}
```